app/Http/Controllers/TrainingDash.php

```diff
@@ -269,7 +269,8 @@ public function saveNewTicket(Request $request) {
             'date' => 'required',
             'start' => 'required',
             'end' => 'required',
-            'duration' => 'required'
+            'duration' => 'required',
+            'score' => ['required', 'digits_between:1,5']
         ]);
```

Comment:
Please add validation here to ensure that movements is numeric. Per the API errata, movements requires an integer.